src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -282,6 +282,8 @@ trait OptimisticTransactionImpl extends TransactionalWrite with SQLMetricsReport
   def commit(actions: Seq[Action], op: DeltaOperations.Operation): Long = recordDeltaOperation(
       deltaLog,
       "delta.commit") {
+    commitStartNano = System.nanoTime()
```

Comment:
Why have this variable on the class level? I would make a local `val` here, more clarity and less mutable variables :)

Reply:
Totally agreed with you. However, passing it through multiple methods looks ugly as well :)